packaging/linux/isle_decomp.desktop.in

```diff
@@ -0,0 +1,33 @@
+[Desktop Entry]
```

Comment:
file name should be isledecomp, the others too

Reply:
Done! It'll be in my next commit.